docs/configuring.asciidoc

```diff
@@ -25,7 +25,28 @@ include::./copied-from-beats/outputconfig.asciidoc[]
 
 include::./configuring-ingest.asciidoc[]
 
+[[configuration-ssl-landing]]
```

Comment:
I'm not seeing this in http://apm-server_2715.docs-preview.app.elstc.co/guide/en/apm/server/master/configuring-howto-apm-server.html -- it's still showing "SSL output settings". I suppose the list above needs to be updated.

Reply:
Oops, thanks!